src/relay/quantize/realize.h

```diff
@@ -0,0 +1,80 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*!
+ *
+ * \file realize.h
+ *
+ * \brief Header of definitions for realization
+ *
+ */
+#ifndef TVM_RELAY_QUANTIZE_REALIZE_H_
+#define TVM_RELAY_QUANTIZE_REALIZE_H_
+
+#include <tvm/relay/transform.h>
+
+namespace tvm {
+namespace relay {
+namespace quantize {
+
+class QRealizeExpr;
```

Comment:
Do we need these declarations as we define things below?

Reply:
Nope, I removed them in the fourth commit.